tests/test_bvmf_calendar.py

```diff
@@ -0,0 +1,72 @@
+from unittest import TestCase
+import pandas as pd
+
+from .test_trading_calendar import ExchangeCalendarTestBase
+from trading_calendars.exchange_calendar_bvmf import BVMFExchangeCalendar
+
+
+class BVMFCalendarTestCase(ExchangeCalendarTestBase, TestCase):
+
+    answer_key_filename = 'bvmf'
+    calendar_class = BVMFExchangeCalendar
+
+    MINUTE_INDEX_TO_SESSION_LABELS_START = pd.Timestamp('2011-01-05', tz='UTC')
+    MINUTE_INDEX_TO_SESSION_LABELS_END = pd.Timestamp('2011-04-05', tz='UTC')
+
+    HAVE_EARLY_CLOSES = False
+
+    DAYLIGHT_SAVINGS_DATES = ['2017-02-17', '2017-10-16']
+
+    # Open from 10am to 5:30pm
+    MAX_SESSION_HOURS = 7.5
+
+    def test_normal_year(self):
+        expected_holidays_2017 = [
+            pd.Timestamp('2017-01-25', tz='UTC'),  # Sao Paolo City Anniversary
+            pd.Timestamp('2017-02-27', tz='UTC'),  # Carnival
+            pd.Timestamp('2017-02-28', tz='UTC'),  # Carnival
+            pd.Timestamp('2017-04-14', tz='UTC'),  # Good Friday
+            pd.Timestamp('2017-04-21', tz='UTC'),  # Tiradentes Day
+            pd.Timestamp('2017-05-01', tz='UTC'),  # Labor Day
+            pd.Timestamp('2017-06-15', tz='UTC'),  # Corpus Christi Day
+            pd.Timestamp('2017-09-07', tz='UTC'),  # Independence Day
+            pd.Timestamp('2017-10-12', tz='UTC'),  # Our Lady of Aparecida Day
+            pd.Timestamp('2017-11-02', tz='UTC'),  # All Souls Day
+            pd.Timestamp('2017-11-15', tz='UTC'),  # Proclamation of the
+                                                   #  Republic Day
+            pd.Timestamp('2017-11-20', tz='UTC'),  # Black Consciousness Day
+            pd.Timestamp('2017-12-25', tz='UTC'),  # Christmas Day
+            pd.Timestamp('2017-12-29', tz='UTC'),  # Day before New Years
```

Comment:
We're not checking  New Year's Day and Constitutional Revolution Day here because they fall on the weekend, is there another year where these fall better? Or maybe we could just find a year in which they're observed and check multiple years here?

Reply:
good idea, I added some other years.